_includes/analytics.html

```diff
@@ -24,3 +24,57 @@
 <script src="//scrapinghub.refersion.com/tracker/v3/pub_187fec375fa525a197a6.js"></script>
 <script>_refersion();</script>
 <!-- REFERSION TRACKING: END -->
+
+<!-- Google Ads: 973726542 Re-targeting code-->
+<script async src="https://www.googletagmanager.com/gtag/js?id=AW-973726542"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'AW-973726542');
+</script>
+
+<!-- Quora Pixel Code (JS Helper) -->
+<script>
+!function(q,e,v,n,t,s){if(q.qp) return; n=q.qp=function(){n.qp?n.qp.apply(n,arguments):n.queue.push(arguments);}; n.queue=[];t=document.createElement(e);t.async=!0;t.src=v; s=document.getElementsByTagName(e)[0]; s.parentNode.insertBefore(t,s);}(window, 'script', 'https://a.quora.com/qevents.js');
+qp('init', 'd3cdc58cc507450a97a918ffd749d99a');
+qp('track', 'ViewContent');
+</script>
+<noscript><img height="1" width="1" style="display:none" src="https://q.quora.com/_/ad/d3cdc58cc507450a97a918ffd749d99a/pixel?tag=ViewContent&noscript=1"/></noscript>
```

Comment:
We can remove this line, It's an image version of above code.. Sorry, I missed to remove it from the sheet.